src/content/docs/tutorial-create-alerts/create-new-relic-alerts.mdx

```diff
@@ -35,13 +35,14 @@ An **incident** is an individual event that details a symptom of a problem, whil
 
 In New Relic, you most often manage your alerts from [**one.newrelic.com > Alerts & AI**](https://one.newrelic.com/alerts-ai). This section of the platform has an **Overview** page that gives you visual breakdowns of all your alerts, as well as an **Alert conditions (policies)** page which gives you a list of all your alerting policies as well as a place to create new ones.
 
-This doc series deals primarily with how to set up basic alerting to give you a solid foundation for getting started and learning exploring the aspects of alerting that are most relevant to your organization. For a full overview of all our alerting features, check out our [alerting docs](/docs/alerts-applied-intelligence/overview/).
-
 ## What's in this series? [#alerts-series]
 
+This doc series deals primarily with how to set up basic alerting to give you a solid foundation for creating alerts using the any of the three most common creation options. It also teaches you how to manage the quality of your alerts, use your created alerts to find information about your system, and improve your stack using the information your alerts have given you.
+
 <DocTiles>
   <DocTile title='Getting started with New Relic alerts' path="/docs/tutorial-create-alerts/create-new-relic-alerts/" label={{text: "You are here", color: "orange"}}> Learn about New Relic alerts</DocTile>
   <DocTile title='Create alerts from charts' path="/docs/tutorial-create-alerts/create-chart-alerts/"> Learn how to create alerts from a chart</DocTile>
   <DocTile title='Create NRQL alerts' path="/docs/tutorial-create-alerts/create-nrql-alerts/"> Learn how to create alerts from NRQL queries</DocTile>
   <DocTile title='Improve your stack with alerts' path="/docs/tutorial-create-alerts/improve-with-alerts/"> Learn how to use alerts to improve your system</DocTile>
+  <DocTile title='Manage your alert quality' path="/docs/tutorial-create-alerts/manage-alert-quality/"> Learn how to manage your alerts to provide you with maximum value</DocTile>
```

Comment:
`to provide you with maximum value` doesn't really tell me much or have much specific meaning. What's the core user value? How would you describe it to a friend or colleague over a beer?